docs/manual/getting_started/benchmarking.rst

```diff
@@ -1,124 +1,76 @@
+tool and the examples perform the benchmarking using sequences of octets with different 
+parameterized sizes.
 
-If the data is not exchanged on the network between the two ddsperf
-instances, it is likely that |var-project| has not selected the
-the appropriate network card on both machines or a firewall in-between is
-preventing communication.
+.. important::
 
-|var-project| automatically selects the most available network interface.
-This behavior can be overridden by changing the configuration file. (see
-section :ref:`test your installation <test_your_installation>` for more details) .
+    The ``throughput`` and ``latency`` examples are not suitable as general DDS 
+    benchmarking tools as they sacrifice performance for simplicity. You cannot compare
+    test results from these tools with results from other tools.
 
-When running the previous scenario on a local machine, this test ensures
-the loop-back option is enabled.
+.. include:: installation/test_installation.part.rst
 
 Measuring latency
 -----------------
 
-To measure latency between two different applications, you need to run
-two instances of the ``ddsperf`` tool and instruct one of them to endorse
+To measure latency between two different applications, you run
+two instances of the ``ddsperf`` tool. The first instance has
 the role of a *sender* that sends a given amount of data (a sequence of
-octets) at a given rate, and the other instance takes the role of
+octets) at a given rate. The second instance has the role of
 *receiver* that sends back the same amount of data to the sender in a
 Ping-Pong scenario. The sending action is triggered by the **Ping**
 option. The receiving behavior is triggered by the **Pong** action. The
 sender measures the roundtrip time and computes the latency as half of
 the roundtrip time.
 
-The Ping-Pong scenario avoids clock desynchronization issues that might
+The Ping-Pong scenario avoids clock desynchronization issues that can
 occur between two machines that do not accurately share the same
 perception of the time in the network.
 
 .. image:: /_static/gettingstarted-figures/4.3-1.png
    :align: center
 
 To differentiate  the two operational modes, the ``ddsperf`` tool can
-operate either in a **Ping mode** or in a **Pong mode**.
+operate either in a **Ping mode** or in a **Pong mode**. 
 
-To run this scenario, open two terminals (e.g., on Linux-like OSs) and run
-the following commands in either of the terminals. The graphical
-*python-based* alternative is also noted.
+To run the Ping-Pong scenario (with default values):
 
-.. code-block:: bash
+1. Open two terminals (for example, on Linux-like OSs). 
+2. In the first terminal, run the following command: 
+  
+ .. code-block:: bash
 
     ddsperf ping
     cyclonedds performance ping
 
-Input this command in another terminal:
+3. In the second terminal run the following command:
 
-.. code-block:: bash
+ .. code-block:: bash
 
     ddsperf pong
     cyclonedds performance pong

```

Comment:
I think this second line is output from the program to the command line, so maybe leave that out.